src/schemas/json/substreams-manifest-schema.json

```
@@ -0,0 +1,173 @@
+{
+  "name": "Substreams Manifest Schema",
+  "description": "A Schema to support in making Substreams manifest files",
+  "type",: "object",
+  "properties" : {
+    "specVersion" : {
+      "enum" : ["v0.1.0"]
+    },
+    "package" : {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "pattern": "^([a-zA-Z][a-zA-Z0-9_]{0,63})$"
+        },
+        "version": {
+          "type": "string",
+          "pattern": "^\\w[a-zA-Z@#0-9.]*$"
+        },
+        "url": {
+          "type": "string"
+        },
+        "doc": {
+          "type": "string"
+        }
+      },
+      "required": ["name", "version"],
+      "additionalProperties": false
+    },
+    "imports": {
+      "type": "object",
+      "properties": {
+      },
+      "propertyNames": {
+        "pattern": "^[A-Za-z_][A-Za-z0-9_]*$"
+      },
+      "additionalProperties": {
+        "type": "string",
+        "pattern": "^\\w[a-zA-Z@#0-9.]*$"
+      },
+      "minProperties": 1
+    },
+    "protobuf": {
+      "type": "object",
+      "properties": {
+        "files": {
+          "type": "array",
```

```suggestion
          "type": "array",
          "uniqueItems": true,
```

Required fix: add `uniqueItems` key